content/includes/experimental.md

```diff
@@ -1,7 +0,0 @@
-Experimental features provide early access to future product functionality.
```

Comment:
oh! interesting; thought we still would have this used somewhere, but maybe we don't 🤔